docs/articles/documentation/guides/advanced-guides/speed-up-test-execution.md

```diff
@@ -103,3 +104,37 @@ This approach ensures that the page model object is created only once per test r
 > Important! If you export the page model class and create an instance in every test file, JavaScript creates a new page object per test file. This increases test execution time and memory consumption.
 
 For more on page models, read [Page Model](../../guides/concepts/page-model.md).
+
+## Enable Server-Side Caching
+
+> Important! Server-side caching is **experimental**.
+>
+> Don't use caching if you run into compatibility issues with your tests.
+
+TestCafe can store webpage assets (like styles, scripts, and images) in its server cache. When TestCafe visits the page again, it pulls assets from the cache instead of requesting them from server.
```

Comment:
+ TestCafe doesn't visit pages, the browser does
+ 'server cache' and 'server-side cache' are two different things, and the former expression can be misinterpreted

```suggestion
The TestCafe proxy can cache webpage assets (such as stylesheets, scripts, and images) and retrieve them from its cache the next time it accesses the webpage.
```